transaction_server.go

```diff
@@ -489,34 +512,44 @@ func (ts TransactionServer) CancelSetSell(params ...string) string {
 	return "1"
 }
 
-// Params: user, filename
-// Purpose: Print out the history of the users transactions to the user specified file
-// Post-Condition: The history of the user's transaction are written to the specified file.
+// DumpLogUser Print out the history of the users transactions
+// to the user specified file
 func (ts TransactionServer) DumpLogUser(params ...string) string {
-	user := params[0]
-	filename := params[1]
+	filename := params[0]
+	user := params[1]
 	ts.Logger.DumpLog(filename, user)
 	return "1"
 }
 
-// Params: filename
-// Purpose: Print out to the specified file the complete set of transactions that have occurred in the system.
-// Pre-Condition: Can only be executed from the supervisor (root/administrator) account.
-// Post-Condition: Places a complete log file of all transactions that have occurred in the system into the file specified by filename
+// DumpLog prints out to the specified file the complete set of transactions
+// that have occurred in the system.
+// Can only be executed from the supervisor (root/administrator) account.
 func (ts TransactionServer) DumpLog(params ...string) string {
 	filename := params[0]
 	ts.Logger.DumpLog(filename, nil)
 	return "1"
 }
 
-// Params: user
-// Purpose: Provides a summary to the client of the given user's transaction history and the current status of their accounts as well as any set buy or sell triggers and their parameters
-// Post-Condition: A summary of the given user's transaction history and the current status of their accounts as well as any set buy or sell triggers and their parameters is displayed to the user.
+// DisplaySummary provides a summary to the client of the given user's
+// transaction history and the current status of their accounts as well
+// as any set buy or sell triggers and their parameters.
 func (ts TransactionServer) DisplaySummary(params ...string) string {
 	panic("not implemented")
 }
 
-func (ts TransactionServer) sellExecute(trigger *Trigger) {
+// getRunningTrigger returns a point to the running trigger that corresponds
+// to the given user, stock combo.
+// If there is not a matching running trigger, returns nil
+func (ts TransactionServer) getRunningTrigger(user string, stock string) *triggers.Trigger {
```

Comment:
Maybe we can use a map and hash the string "user,stock"? This is probably fine for now tho.